Ladybird/cmake/ResourceFiles.cmake

```diff
@@ -78,7 +78,7 @@ set(CONFIG_RESOURCES
     BrowserAutoplayAllowlist.txt
     BrowserContentFilters.txt
 )
-list(TRANSFORM CONFIG_RESOURCES PREPEND "${LADYBIRD_SOURCE_DIR}/Base/home/anon/.config/")
+list(TRANSFORM CONFIG_RESOURCES PREPEND "${LADYBIRD_SOURCE_DIR}/Base/res/ladybird/")
```

Comment:
these should be merged into the "web resources" list, in both cmake and gn

Reply:
though actually, we want these to be installed into `XDG_CONFIG_HOME`, so that users can edit them at their leisure. Not sure how to do that properly from cmake, seems to be a packaging problem.

Reply:
Then my idea is to keep these as a resource, but as a template for the default config. Ladybird then detects a first-run and I guess copies them to its system dependent configuration directory?

Reply:
I've decided to revert this change and make a more permanent one later